### DISCUSSION TITLE
What do you think about fluid-tailwind?

### DISCUSSION THREAD

**Discussion by frederikhors** [tailwindlabs/tailwindcss#12989] [Ideas]
What do you think about https://github.com/barvian/fluid-tailwind/?

Why not in Tailwind directly?

**Comment by adrienpoly** [comment]
I would love a built in solution for fluid responsive fonts in TW

For now I use RFS like that https://github.com/tailwindlabs/tailwindcss/discussions/8087#discussioncomment-9304574

Not a big fan of `~` (always a hard character to type for me)

I preferred the initial proposal of @MartijnCuppens  https://github.com/tailwindlabs/tailwindcss/discussions/12238

something like `fluid:text-4xl`
